src/public/packages/backpack/crud/js/selects.js

```diff
@@ -0,0 +1,223 @@
+let refreshOptionList = function (element, $field, $refreshUrl) {
```

Comment:
This is a promise that is called upon on the fly entity created to refresh select options. Why ? Imagine in this crud, your options are scoped by `active()`. You create a new entity, but you don't set it active. That option should not be available for user selection. Thus we reload the options and go on from there ...